site/content/en/docs/manual/advanced/formats/format-camvid.md

```diff
@@ -22,13 +22,23 @@ taskname.zip/
 |   └── image2.png
 └── <any_subset_name>.txt
 
-# labelmap.txt
+# label_colors.txt (with color value type)
+# if you want to manually set the color for labels, configure label_colors.txt as follows:
 # color (RGB) label
 0 0 0 Void
 64 128 64 Animal
 192 0 128 Archway
 0 128 192 Bicyclist
 0 128 64 Bridge
+
+# label_colors.txt (without color value type)
```

Comment:
@zhiltsov-max 
I add example code about label_colors.txt without color value.
Please check if it is easy to understand through the example code.

Thank you for your suggestion about adding example code.